tests/FunctionalTest/ContainerInteractionTest.php

```diff
@@ -69,7 +69,9 @@ public function phpdi_aliases_can_reference_symfony_entries()
         /** @var Container $phpdiContainer */
         $phpdiContainer = $container->getFallbackContainer();
 
-        $phpdiContainer->set('foo', \DI\get('class2'));
+        $ref = \DI\get('class2');
+        $ref->setName('foo');
+        $phpdiContainer->set('foo', $ref);
```

Comment:
Did you have to do this? Because it looks like a regression in PHP-DI 6 to me 🤔

Reply:
Yes, for me its a regression in PHP-DI 6, the \DI\get returns a reference and not an helper, and so the entry name will never injected into it. (In PHP-DI 5 its an helper).

But I use PHP-DI only since few months, (it'is a pretty tool). So my experience with this project is limited and I did not have time to see if that was the expected behavior or not for PHP-DI 6.

Reply:
Thank you I have created an issue to check this in PHP-DI itself. It should not block this PR though.